test/parallel/test-fs-syncwritestream.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const spawn = require('child_process').spawn;
+const stream = require('stream');
+const fs = require('fs');
+const path = require('path');
+
+// require('internal/fs').SyncWriteStream is used as a stdio implementation
+// when stdout/stderr point to files.
+
+if (process.argv[2] === 'child') {
+  console.log(JSON.stringify([process.stdout, process.stderr].map((stdio) => ({
```

Comment:
Nit: Perhaps a note here quickly indicating that the `console.log` is intentional and part of the test? There are plenty of places around in the tests where `console.log` is used without serving much purpose.